includes/QueryPrinterFactory.php

```diff
@@ -7,21 +7,21 @@
 /**
  * Factory for "result formats", ie classes implementing QueryResultPrinter.
  *
- * @since 1.9
+ * @since 2.5 (since 1.9, renamed in 2.5)
  *
  * @ingroup SMW
  *
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-final class FormatFactory {
+final class QueryPrinterFactory {
```

Comment:
I think this should go into `SMW\Query\ResultPrinterFactory` (with printers being move to `SMW\Query\ResultPrinters`). I'd like to minmize arbitrary access to namespaced classes from the outside therefore this should be accessible either via `ApplicationFactory::getInstance()->getQueryPrinterFactory` or  `ApplicationFactory::getInstance()->getQueryFactory()->getPrinterFactory`.